kong/plugins/rate-limiting/handler.lua

```diff
@@ -72,7 +80,7 @@ function RateLimitingHandler:access(conf)
 
   -- Consumer is identified by ip address or authenticated_credential id
   local identifier = get_identifier(conf)
-  local api_id = ngx.ctx.api.id
+  local api_id = get_api_id(conf)
```

Comment:
Is there really a measurable amount of function overhead? I certainly prefer to have a function whose name clearly defines its purpose rather than a standalone complex expression whenever possible.